### PR TITLE
Clarifying wireless operation

### PR DESCRIPTION
**Pull request by DustinGrau**
Also, removed experimental code that was not supposed to go live just yet...not a big issue, but not ready for prime time.

### CODE REVIEW

**Comment by DustinGrau** [comment]
Open to anyone to review :)

File: ATTENUATOR_FLASHING.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ This device supports Over-The-Air (OTA) updates for firmware, meaning you will n
 
 📝 **Tip:** Before proceeding, be sure to use a high-quality USB cable which supports data transfer. Some cheap cables may only support charging (not data), or not fully support the power requirements of the device. If you have successfully flashed your ESP32 device and do not see the available WiFi access point, try plugging your USB cable directly into the Talentcell battery or try another USB port on your computer. In rare cases the USB port and/or cable cannot supply enough voltage to run the ESP32's WiFi radio.
 
-### First-Time Upload
+### ESP32: First-Time Upload
+
+If you do not see a WiFi network for your Proton Pack, the device has likely not been flashed with the GPStar firmware to enable that feature. You will need to upload the software using a USB cable via either the 2 methods listed here.
```
Comment (nomakewan) [inline ATTENUATOR_FLASHING.md:21]: Should be either _of_ the 2 methods

File: WIRELESS_OPERATION.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ This guide will cover the web interface available via the Attenuator or Wireless
 
 Please see the [ATTENUATOR_FLASHING](ATTENUATOR_FLASHING.md) guide for details on compiling and/or uploading software to your Wireless Adapter controller.
 
+## Hardware Synchronization
+
+In order to view the state of the pack and control it remotely, the two devices must be physically connected and have been sychnronized via the built-in software. On the ESP32 are 2 on-board LED's which will show the current status. A red LED indicates the device is powered and should be accessible via WiFi. A blue LED indicates the device has successfully synchronized with the Proton Pack.
```
Comment (nomakewan) [inline WIRELESS_OPERATION.md:11]: No apostrophe on LEDs

File: WIRELESS_OPERATION.md [hunk 3]
```diff
@@ -30,6 +34,10 @@ This section allows full control of the master (overall) volume and to mute/unmu
 
 Controls will be made available on a per-action or per-state basis. Shown here, the pack and wand are both in an Idle state while in the "Original" mode which allows the pack to be turned on/off remotely. The options to remotely vent or to "Attenuate" are only enabled when the devices are in a specific state.
 
+**Vent:** This can only be triggered when in the Super Hero mode and while the pack is Powered.
```
Comment (nomakewan) [inline WIRELESS_OPERATION.md:37]: Is "Powered" something specific in the GPStar code? Does this mean "On" as in "MODE_ON"? Perhaps better to clarify, for example, "and while the cyclotron is spinning?" or "and while the cyclotron is active?" or some other more descriptive term?

**Review by nomakewan — CHANGES_REQUESTED** [review]
Looks good overall, just a few little nitpicks I noticed.

File: WIRELESS_OPERATION.md (hunk 3 above)
Comment (nomakewan) [reply WIRELESS_OPERATION.md:37]: Ah gotcha, so it's referring to something in the Attenuator's WiFi GUI? Thank you for the clarification, that makes a lot more sense.

File: WIRELESS_OPERATION.md (hunk 2 above)
Comment (nomakewan) [reply WIRELESS_OPERATION.md:11]: Yeah, a quick search shows 3 in comments in Attenuator.ino and 1 in the Attenuator's Header.h. But that's it from what I can tell.

**Pull request merged**